tests/forms_tests/field_tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+from django.forms import Form
+from django.test import SimpleTestCase
+
+
+class BaseFieldTest(SimpleTestCase):
```

Comment:
I'd rather call this something like `FormFieldAssertionsMixin` and have it inherit `object`.